src/components/Homepage/LendByCategory/LoanCategoriesSection.vue

```diff
@@ -172,6 +184,7 @@ export default {
 				}).then(({ data }) => {
 					const channelLoans = _get(data, 'lend.loanChannelsById')[0];
 					this.categoriesWithLoans.push(channelLoans);
+					this.$emit('loans-loaded');
```

Comment:
You're not listening to this event anywhere. Im assuming you want to listen to this on the homepage and fire `trackLoansLoaded`

Reply:
The listener for this lives alongside the missing ref on the loan categories row...

Reply:
listener is now in place